src/domain/entities/__init__.py

```
@@ -0,0 +1,4 @@
+"""����ƣƣn������"""
```

Ruff (0.12.2)

stream did not contain valid UTF-8

(E902)